synapse-modules/alkemio_room_control.py

```
@@ -34,77 +34,158 @@
+                        visibility_event = await state_storage.get_current_state_event(
+                            room_id, ALKEMIO_VISIBILITY_EVENT, ""
+                        )
+                        if visibility_event:
+                            visible = visibility_event.content.get("visible")
+                            logger.debug(
+                                "Sync filter: room %s visibility=%s",
+                                room_id, visible,
+                            )
+                            if visible is False:
+                                hidden_room_ids.add(room_id)
+                    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)